healsparse/_version.py

```diff
@@ -1 +1 @@
-__version__ = '1.2.8'
+__version__ = '1.2.9'
```

Comment:
bumping to 1.3.0 since there is a new feature would be sensible, though not required.